crud.py

```diff
@@ -0,0 +1,61 @@
+from sqlalchemy.orm import Session
+
+import schemas
+import models
+
+
+def get_all_authors(db: Session):
+    return db.query(models.Author).all()
+
+
+def get_author_by_id(db: Session, author_id: int) -> models.Author | None:
+    return db.query(models.Author).filter(models.Author.id == author_id).first()
+
+
+def create_author(db: Session, author: schemas.AuthorCreate) -> models.Author:
+    db_author = models.Author(
+        name=author.name,
+        bio=author.bio
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+
+    return db_author
+
+
+def get_all_books(
+        db: Session,
+        author_id: str | None = None,
+        skip: int | None = None,
+        limit: int | None = None
+) -> list[models.Book]:
+    queryset = db.query(models.Book)
+
+    if author_id:
```

Comment:
Ensure that `author_id` is correctly passed as an integer. The current type hint suggests it might be a string, which could lead to issues when filtering by `Author.id`.